ui/apps/platform/src/Containers/Policies/PatternFly/policies.utils.ts

```diff
@@ -79,17 +79,6 @@ export function getSearchStringForFilter(s: SearchFilter): string {
     );
 }
 
-/*
```

Comment:
This is the only change that is more than type-level in this PR. This function will be used in the Violations search changes so it was moved to a higher level searchUtils that isn't policy specific.